rascal-lsp/src/main/rascal/lang/rascal/tests/rename/Fields.rsc

```diff
@@ -81,6 +81,19 @@ test bool commonKeywordFieldsSameType() = testRenameOccurrences({0, 1},
     decls = "data D (set[loc] foo = {}, set[loc] baz = {})= d();"
 );
 
+test bool sameNameFields() = testRenameOccurrences({0, 2}, "
+    'D x = d(8);
+    'int i = x.foo;
+", decls = "
+    'data D = d(int foo);
+    'data E = e(int foo);
+");
+
+test bool sameNameFieldsDisconnectedModules() = testRename({
+    byText("A", "data D = d(int foo);", {0})
+  , byText("B", "data E = e(int foo);")
+});
+
```

Comment:
Should we add a test for equal field names in different constructors of a single ADT?

Reply:
We already have [this](https://github.com/usethesource/rascal-language-servers/pull/487/files#diff-93a989963a285dc6bd654287ba732805ffe9038a045761115ee7b35b86e3f00aL52-L56). Does this cover the cases that you are referring to?